hbase_sql/src/main/java/com/splicemachine/hbase/HBaseRegionLoads.java

```diff
@@ -234,6 +234,11 @@ private static String tableForRegion(String regionName){
     public Collection<PartitionLoad> tableLoad(String tableName, boolean refresh){
         if (refresh) {
             Map<String, Map<String, PartitionLoad>> loads = cache.get();
+            if (loads == null) {
```

Comment:
Is this related to this PR?

Reply:
Somewhat. Erin hit this problem when she tried to fix a table